web-client/src/ducks/auth/phone/actions.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  phoneAuthTrigger,
+  phoneAuthVerify,
+} from 'src/http/resources/auth/phoneAuth';
+
+import { IOTPAuth, IPhoneNumberAuth } from '../types';
```

Comment:
We had a conversation where we decided to create new issue regarding deciding the best standard for defining the interfaces.
So decision on this will be taken in that issue and will be outside of this PR